definitions/ext-aerospike/definition.yml

```diff
@@ -2,14 +2,21 @@ domain: EXT
 type: AEROSPIKE
 
 synthesis:
-  identifier: aerospike_identifier
-  name: aerospike_hostname
-  encodeIdentifierInGUID: true
-  conditions:
-    - attribute: metricName
-      prefix: aerospike_
-  tags:
-    clusterName:
+  rules:
+  - compositeIdentifier:
```

Comment:
If you change the identifier, you will have a short time of duplicates (one with the old definition and the ones created using the new composition), and you will lose the telemetry associated with the old GUID.

Reply:
Great.. Thank you for that @apoloa , but I am currently running a test case to integrate aerospike technology. Hence, it is fine if I loose any telemetry associated with the old GUID. 

Thank you 😃

Reply:
If you need help w/ setting the right identifiers just ping us on the internal help channel: #help-entity-platform